static_src/components/route_form.jsx

```diff
@@ -155,6 +158,17 @@ export default class RouteForm extends React.Component {
             className={ this.styler('route-form-preview') }
             value={ this.fullUrl }
           ></input>
+          <div style={{ paddingLeft: '.75rem' }}>
+            {(() => {
+              if (this.props.error) {
+                return <FormError message={this.props.error.description} />;
```

Comment:
Yea I think I might amend the naming.

One source of error is when creating a new route. That's very different because it's not attached to a specific route, because the route wasn't created yet, hence doesn't have a guid. This one comes from `props.error`.

The other error is when you try and update/delete an existing route and something goes wrong. This error is associated with a route so we place it on the actual route object.

When both have an error, we give precedence to the specific route error. This is an actual case that can happen where you have both the create new route and edit route forms open and two separate error messages.

Any suggestions on naming that would make it more clear?